logins-api/android/library/build.gradle

```diff
@@ -145,6 +145,6 @@ archivesBaseName = 'logins'
 
 apply from: '../../../publish.gradle'
 ext.configurePublish(
```

Comment:
Hold off on this -- we want to keep the Maven coordinates constant.  In addition, for reasons I don't fully understand, a-c is publishing to `org.mozilla.components:support-base`, like http://maven.mozilla.org/?prefix=maven2/org/mozilla/components/support-base/.  We should follow suit and publish to `org.mozilla.appservices:logins`, but we're not quite ready to do that yet.

Reply:
(Changing the Maven coordinates should be part of #252; there's no advantage, really, until we can control the publishing story more fully.)